plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -1507,6 +1488,15 @@
                         categories="nosql,timeseries,aws">
                     <notAvailableDriver title="%not.available.title" description="%not.available.description" message="%driver.timestream.reason"/>
                 </driver>
+                <driver
```

Comment:
You cannot delete old driver. neo4j_jdbc is used in PRO products for replacement.
Keep old driver